src/lib/components/Header.svelte

```diff
@@ -0,0 +1,24 @@
+<header>
+	<img src="/assets/images/logo.svg" alt="Hogeschool van Amsterdam Logo" />
+	<span>Frontend <br /> Design & Development</span>
+</header>
+
+<style>
+	header {
+		display: flex;
+		justify-content: flex-start;
+		align-items: center;
+        color: var(--color-white);
```

Comment:
Weird spacing?

Reply:
github is just weird, in my code editor it is looking normal